src/gettext_boost.cpp

```diff
@@ -11,6 +11,9 @@
 
    See the COPYING file for more details.
 */
+#if defined(_MSC_VER)
+#pragma warning(disable: 4714)
```

Comment:
It was originally there. I moved it to make it apparent it's fully file-scoped. Note no pop/push either.
The warning occurs during link-time code generation.